### PR TITLE
feat(itemmenu): add ability to refresh library from item context menu

### PR DESCRIPTION
**Pull request by Camc314**
This method does not allow the ability to refresh and replace all images or metadata, if users want to do that, I think it should be done through settings.

### CODE REVIEW

File: components/Item/ItemMenu.vue [hunk 1]
```diff
@@ -56,9 +63,42 @@ export default Vue.extend({
...
+          menuItems.push({
+            title: this.$t('refreshLibrary'),
+            action: async () => {
+              try {
+                await this.$api.itemRefresh.post({
+                  itemId: this.item.Id,
+                  replaceAllImages: false,
+                  replaceAllMetadata: false
+                });
+                this.pushSnackbarMessage({
```
Comment (MrTimscampi) [inline components/Item/ItemMenu.vue:80]: Not sure how necessary this is. Maybe as a "normal" color one, not a success one ?

File: components/Item/ItemMenu.vue [hunk 2]
```diff
@@ -56,9 +63,42 @@ export default Vue.extend({
             }
           });
         }
+
+        if (
+          this.$auth.user?.Policy?.IsAdministrator &&
+          (this.item.Type === 'Folder' || this.item.Type === 'CollectionFolder')
```
Comment (MrTimscampi) [inline components/Item/ItemMenu.vue:69]:
```suggestion
          (['Folder', 'CollectionFolder', 'UserView'].includes(this.item.Type || '')
```

`UserView` is for merged libraries (In jf-web, go to Settings > Home > Library Folders)

**Review by MrTimscampi — CHANGES_REQUESTED** [review]
Two small things, aside from that, LGTM

File: components/Item/ItemMenu.vue (hunk 1 above)
Comment (Camc314) [reply components/Item/ItemMenu.vue:83]: Updated to use the default color 👍

File: components/Item/ItemMenu.vue (hunk 2 above)
Comment (Camc314) [reply components/Item/ItemMenu.vue]: Ah ok, updated 👍

**Review by MrTimscampi — APPROVED** [review]

**Pull request merged**